HW06.Rmd

```diff
@@ -30,34 +30,38 @@ From the RNA-Seq files, you only need the "Name" from one file and the "TPM" col
 
 library(tidyverse)
 
+#To establish a vector with the all the filepaths to read
+H3K27M_filepath <- list.files("RNA_Seq_processed/H3K27M", full.names = TRUE) #for the H3K27 folder
+WT_filepath <- list.files("RNA_Seq_processed/WT", full.names = TRUE) #for the WT folder
+RNA_seq_file <- c(H3K27M_filepath, WT_filepath) #To get a complete vector of all the filepaths
+
+#To establish a vector with just the file names to keep track of where the data came from
 H3K27M_names <- list.files("RNA_Seq_processed/H3K27M")
 WT_names <- list.files("RNA_Seq_processed/WT")
-
+#And also to be able to track which ones where from the WT and H3K27 folders
 H3K27M_names <- gsub(pattern = "\\.sf", H3K27M_names, replacement = "/H3")
 WT_names <- gsub(pattern = "\\.sf", WT_names, replacement = "/WT")
-cnames <- c("gene_id", H3K27M_names, WT_names)
+column_names <- c("gene_id", H3K27M_names, WT_names) #And with this we have the complete list of names for the GBM.transcripts matrix
 
-H3K27M_filepath <- list.files("RNA_Seq_processed/H3K27M", full.names = TRUE)
-WT_filepath <- list.files("RNA_Seq_processed/WT", full.names = TRUE)
-RNA_seq_file <- c(H3K27M_filepath, WT_filepath)
 
+#Function for importing and cleaning the data
 import_clean <- function(x){
   if(x == "RNA_Seq_processed/H3K27M/BT245_C19.sf") {

```

Comment:
Using a function to solve this problem is smart. The only improvement I may suggest is to make it so that the if statement acts on the name of the first element in the list of filenames. That way, you could use this regardless of whether that file has been renamed. Hopefully that makes sense...